content/_changelogs/10.11.0.md

```diff
@@ -0,0 +1,29 @@
+## 10.11.0
+
+_Released 10/25/2022_
+
+**Features:**
+
+- [`cy.origin()`](/api/commands/origin) now supports using `require()` and dynamic `import()` to include
+  dependencies. `Cypress.require()` has been removed. Addresses
+  [#24293](https://github.com/cypress-io/cypress/issues/24293).
+- Adds support for [`cy.session()`](/api/commands/session) to experimental WebKit.
+Ensures that with `experimentalSessionAndOrigin` enabled, tests are still able to run against webkit. Addresses [#24116](https://github.com/cypress-io/cypress/issues/24116), [#23832](https://github.com/cypress-io/cypress/issues/23832).
+- Improves user experience when connecting a project to the Dashboard. Addresses [#23379](https://github.com/cypress-io/cypress/issues/23379), [#23766](https://github.com/cypress-io/cypress/issues/23766), [#23767](https://github.com/cypress-io/cypress/issues/23767).
+  - Users can now connect a project to the Dashboard when logging in from the Cypress Launchpad after a testing type has been selected.
+  - Users will be prompted to record runs to the Dashboard from both the Cypress App and Cypress Launchpad if a project is connected to the Dashboard and has no recorded runs.
+
+**Bugfixes:**
+
+- Fixed issue connecting to the cloud when a self-signed cert was in the cert chain. Fixes [#24298](https://github.com/cypress-io/cypress/issues/24298).
+- The way that Cypress handles [test isolation](/guides/core-concepts/writing-and-organizing-tests#Test-Isolation) has changed.  The previous modes of `legacy` and `strict` have been replaced with `on` and `off`.  Fixes [#24206](https://github.com/cypress-io/cypress/issues/24206).
```

Comment:
I feel like there should be an `Breaking (Experimental)` section. Thoughts?

Reply:
@mschile @mjhenkes I looked through the other [changelogs](https://docs.cypress.io/guides/references/changelog) and could not find an example of a `Breaking (Experimental)` section.  Have we done this before?  Should that section go at the end since it is "experimental"?

Reply:
I don't think it has been done but it seems like good information. Removing cypress.require in favor of require and import would also be an experimental breaking change

Reply:
@mjhenkes @mschile Added an "Experimental Breaking Changes" section to the end and moved the three items mentioned that that section.  Let me know what you think.